src/components/common/CountDownTimer.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div />
+</template>
+
+<script>
+import VueTimers from 'vue-timers/mixin';
+
+export default {
+  name: 'CountDownTimer',
+
+  props: {
+    timeout: {
+      type: Number,
+      default: 30000,
+    },
+    isLocked: {
+      type: Boolean,
+      default: false,
+    },
+    counter: {
+      type: Number,
+      default: 0,
+    },
+  },
+
+  computed: {
+    initialCounter() {
+      return this.timeout / 1000;
```

Comment:
why computed? why not simple `const initialCounter = this.timeout / 1000`?
we don't need to be this reactive and it used only in one place